parallel-consumer-core/src/test/java/io/confluent/parallelconsumer/ParallelEoSStreamProcessorNonRunningTest.java

```diff
@@ -0,0 +1,46 @@
+package io.confluent.parallelconsumer;
+
+/*-
+ * Copyright (C) 2020-2023 Confluent, Inc.
+ */
+
+import io.confluent.parallelconsumer.internal.TestParallelEoSStreamProcessor;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.kafka.clients.consumer.OffsetResetStrategy;
+import org.apache.kafka.clients.consumer.MockConsumer;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Tests that the protected and internal methods of
+ * {@link io.confluent.parallelconsumer.internal.AbstractParallelEoSStreamProcessor} work as expected.
+ * <p>
+ *
+ * @author Jonathon Koyle
+ */
+@Slf4j
+class ParallelEoSStreamProcessorNonRunningTest {
```

Comment:
Just IMHO - maybe can be renamed to something along the lines of AbstractParallelEoSStreamProcessor ConfigurationTest ? or ParallelConsumerConfigurationTest ?

Reply:
Renamed to `AbstractParallelEoSStreamProcessorConfigurationTest`